### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by SamuelUlep** [xremap/xremap#627] [Ideas]
I might be just extremely stupid and its on the 'readme.md' of the git repo but skimmed over it or it's on the app itself, but is there an actual wiki for all the various functions that can be done on 'xremap'?

So far from all of my searching for a Linux alternative for XMBC (X-mouse button control), this seems the best alternative that covers most of the probable use case that XMBC offers. my only issue is I looking for any documentation if there are options for XMBC 'Repeatedly while the button is down', 'Sticky (Repeatedly until button is pressed again)', and etc. with repeat delay and stuff.

**Comment by travisbikkle** [comment]
In fact I can't figure out how to run xremap on Ubuntu24.04 because I use wayland and downloaded the xremap extension, and after switching on the xremap in `extensions` app I don't know where to put the config.yml.

Basically I don't know the relationship between the extension and the rust crate.

Do we need both? If not, where to put the config.yml and load it with xremap gnome shell extension?

**Comment by k0kubun** [comment]
Hmm. I thought it was a good idea to enable GitHub Wiki for xremap at first, but then it seemed tedious to review/manage Wiki unlike documents submitted by PRs.

I do agree that the documentation in README is not really easy to browse though. I'd like to suggest adding multiple documentation files that are linked from an index markdown file, maybe under `doc/` directory. I can merge PRs that implement the more readable documentation under `doc/`. Even if it's just a skelton and a bunch of TODOs, it's fine too.
